crates/dpapi-dg/README.md

```diff
@@ -0,0 +1,3 @@
+# DPAPI Devolutions-Gateway
+
+The main purpose of this crate is to obtain the session token from Devolutions-Gateway using standartalone route.
```

Comment:
issue: Already discussed on Slack, but the webapp routes are dedicated to the Devolutions Gateway standalone, and should not be relied upon.

suggestion: If this is just for testing purpose, you may consider support for [tokengen server](https://github.com/Devolutions/devolutions-gateway/tree/master/tools/tokengen). Note that this is a dev tool, and this is not something that we will ever use in production.

question: My understanding is that `dpapi-cli-client` is also a dev tool (testing)?

Reply:
done